src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts

```diff
@@ -1,94 +1,114 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { FilterConfig } from '@hmcts/rpx-xui-common-lib/lib/models';
+import { CaseUserSearchErrorMessage } from 'src/cases/models/case.enum';
+import { StaffDataFilterService } from 'src/staff-administrator/services/staff-data-filter.service';
```

Comment:
Need to replace src with relative routes '../'

Reply:
Also the build is failing

Reply:
Well note and updated.

Reply:
The build is failing as below PR should be merged before this (https://github.com/hmcts/rpx-xui-common-lib/pull/308).